### DISCUSSION TITLE
QUESTION: Where can I find an example of defining a generic job in xml?

### DISCUSSION THREAD

**Discussion by dmitribodiu** [quartznet/quartznet#389] [General]
Can you show me an example of defining a generic job in xml please?

**Comment by robertstefan** [comment]
@dmitribodiu Take a look at [this](https://github.com/quartznet/quartznet/blob/master/src/Quartz.Server/quartz_jobs.xml) example.

**Comment by cknightdevelopment** [comment]
I have a [blog post](http://knightcodes.com/.net/2016/08/15/xml-configuration-for-quartz-net.html) that includes a video & source code instructing how to configure jobs/triggers via XML.
